wallet-server-ui/src/app/app-routing.module.ts

```diff
@@ -1,29 +1,46 @@
-import { NgModule } from '@angular/core'
-import { CommonModule } from '@angular/common'
-import { RouterModule, Routes } from '@angular/router'
+import { NgModule } from '@angular/core';
```

Comment:
I would prefer you change your local to not add the ';' as we don't use them across files.